tests/unit/binary-operator-printers/index.test.js

```diff
@@ -1,4 +1,4 @@
-const binaryOperatorPrinters = require('../../src/binary-operator-printers');
+const binaryOperatorPrinters = require('../../../src/binary-operator-printers');
```

Comment:
single quotes here?

Reply:
this is our tests with our `.prettierrc` the default config is only for `tests/config`